src/app/Components/Details/QuickAccess/SharedPlan/SharedPlansList/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import findSharedPlans from '../../../../../Gateways/FindSharedPlans';
+import styles from './SharedPlansList.module.scss';
+
+const SharedPlansList = ({ customerId }) => {
+  const [state, setState] = useState({
+    plans: [],
+    loading: true,
+    error: null
+  });
+
+  useEffect(() => {
+    const fetch = async () => {
+      try {
+        const plans = await findSharedPlans({ customerId });
+        setState({ plans, loading: false });
+      } catch (error) {
+        setState({ plans: [], error, loading: false });
+      }
+    };
+
+    fetch();
+  }, [customerId]);
+
+  if (state.plans.length === 0) {
+    return null;
+  }
+
+  return (
+    <div className={styles.wrapper}>
+      <h4 className={styles.heading}>Existing plans</h4>
+      <ul>
+        {state.plans.map(({ id, location }) => (
+          <li key={id}>
+            <a href={location}>{id}</a>
```

Comment:
Also, not entirely sure what the location is, but it takes me to baseURL/<id> instead of baseURL/plans/<id>

Reply:
It's taken from an environment variable so we should be fine to just include the `/plans` as a part of that base URL?

Reply:
That's a good point!